README.md

```diff
@@ -1,7 +1,9 @@
+
 # AssignmentDesignPatterns
 
 Design pattern identified in the CellPhone Program
 
+
 1) Factory Pattern
 
 2) Builder Pattern
```

Comment:
HOw would you use builder and factory in the same example?

Reply:
I'm not implementing both of them in the same example. I found both the pattern is applicable in my program but I am only going to use the Factory Pattern.